output/md/md.go

```diff
@@ -588,6 +593,24 @@ func (m *Md) makeTableTemplateData(t *schema.Table) map[string]interface{} {
 		columnsData = append(columnsData, data)
 	}
 
+	// Viewpoints
+	viewpointsData := [][]string{
+		[]string{
+			m.config.MergedDict.Lookup("Name"),
+			m.config.MergedDict.Lookup("Definition"),
+		},
+		[]string{"----", "----------"},
+	}
+
+	for _, v := range t.Viewpoints {
+		data := []string{
+			fmt.Sprintf("[%s](viewpoint-%d.md)", v.Name, v.Index),
+			v.Desc,
```

Comment:
I would like to change the contents of the display using `showOnlyFirstParagraph`

Reply:
@k1LoW 
Alright! I fixed it 👍